### ISSUE TITLE
Change h3 to h2 for better semantic structure

### ISSUE THREAD

**Issue by Eziokwumgbeokwere** [Eziokwumgbeokwere/business-card#2]
The heading for "Eziokwu Mgbeokwere" is currently an h3 tag. It should be changed to h2 for better semantic structure and accessibility.
